tests/integration/steps_double_sign.go

```diff
@@ -1,15 +1,6 @@
 package main
 
-// simulates double signing on provider and vsc propagation to consumer chains
-//
-// Note: These steps would be affected by slash packet throttling, since the
-// consumer-initiated slash steps are executed after consumer-initiated downtime
-// slashes have already occurred. However slash packet throttling is
-// psuedo-disabled in this test by setting the slash meter replenish
-// fraction to 1.0 in the config file.
-//
-// only double sign on provider chain will cause slashing and tombstoning
-func stepsDoubleSign(consumer1, consumer2 string) []Step {
+func stepsDoubleSign(consumerName string) []Step {
```

Comment:
```suggestion
func stepsProviderDoubleSign(consumerName string) []Step {
```
I thought this was consumer double signing as a quick reader